include/groonga/groonga.h

```diff
@@ -1693,6 +1693,24 @@ grn_column_get_all_index_columns(grn_ctx *ctx,
                                  grn_obj *column,
                                  grn_obj *index_columns);
 
+/**
+ * \brief Delete an object (such as a table or column) identified by the ID from
+ *        the database.
+ *
+ * \note This is an internal API intended for use by mroonga.
+ *
+ * \param ctx The context object.
+ * \param db The target database object.
+ * \param id The ID of the object to be deleted.
+ * \param remove_p A boolean flag indicating the deletion mode:
+ *                 - If \ref GRN_TRUE, the function removes the relation between
```

Comment:
```suggestion
 *                 - If `true`, the function removes the relation between
```

`false`の方と比較しやすいように、「関連を消してキャッシュも消すよ」じゃなくて「キャッシュをクリアするだけじゃなくて関連も消すよ」という順番の方がいいんじゃないかな。

Reply:
fix: [0854ceb](https://github.com/groonga/groonga/pull/2009/commits/0854ceb25129ae25f2181537e312582b4a5ca725) 確かに対になっている方がわかりやすいので修正しました。